polynote-spark/src/test/scala/polynote/kernel/interpreter/python/PySparkInterpreterSpec.scala

```diff
@@ -205,11 +205,8 @@ class PySparkInterpreterSpec extends FreeSpec with InterpreterSpec with Matchers
       } catch {
         case err: Throwable =>
           err shouldBe a[RuntimeException]
-          err.getMessage should include ("AnalysisException: 'Path does not exist:")
-          err.getCause shouldBe a[RuntimeException]
```

Comment:
Is there no longer a cause for this error?

Reply:
Spark 3 kind of messed up the logic for getting exceptions and causes. It's no longer possible, because they install their own exception handler thing into py4j, and then wrap various types of exception into Python data structures (discarding the original JVM exception).

Reply:
So, this test doesn't pass for Spark 3, and it just didn't seem all that important that the exceptions were structured in exactly this way.

Reply:
Ah, gotcha. That's annoying. Mind pointing me somewhere I can learn more about how they changed the py4j error handling stuff?